src/cmap/test/integration.rs

```diff
@@ -26,7 +27,7 @@ async fn acquire_connection_and_send_command() {
     let pool_options = ConnectionPoolOptions::from_client_options(&client_options);
 
     let pool = ConnectionPool::new(client_options.hosts[0].clone(), Some(pool_options));
-    let mut connection = pool.check_out().unwrap();
+    let mut connection = RUNTIME.block_on(pool.check_out()).unwrap();
```

Comment:
Since this is an async function (with the annotations to start the runtime), you can actually `await` here.